<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>552</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Установка точки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="5">
         <widget class="PaintGraphicView" name="graphicsView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>CrossCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QListWidget" name="listFilesTabFirst">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="OpenButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Открыть</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BackButton">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Назад</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="CoordinatesLabel">
              <property name="text">
               <string>x: 0.0 y: 0.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Разность координат</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QListWidget" name="listFilesFirstTabSecond">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="OpenDirForFirstListWidget">
            <property name="text">
             <string>Открыть</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="OpenDirForSecondListWidget">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Открыть</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QListWidget" name="listFilesSecondTabSecond">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="7" column="0" rowspan="2" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="Calculate">
            <property name="text">
             <string>Вычислить</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>814</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PaintGraphicView</class>
   <extends>QGraphicsView</extends>
   <header>paintgraphicview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>Calculate</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>CalculateClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>78</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenQFileDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>517</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>512</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listFilesTabFirst</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MainWindow</receiver>
   <slot>ItemDoubleClicked(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>318</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BackButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>SetBackItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>527</y>
    </hint>
    <hint type="destinationlabel">
     <x>777</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenDirForSecondListWidget</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenDirForSecondListWidgetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>489</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>OpenDirForFirstListWidget</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>OpenDirForFirstListWidgetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>485</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>551</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ItemDoubleClickeded(QListWidgetItem*)</slot>
  <slot>SavePointInFile()</slot>
  <slot>SetNextItem()</slot>
  <slot>SetBackItem()</slot>
  <slot>ItemDoubleClickedTab2FirstListWidget(QListWidgetItem*)</slot>
  <slot>CalculateClicked()</slot>
  <slot>OpenQFileDialog()</slot>
  <slot>OpenDirForFirstListWidgetClicked()</slot>
  <slot>OpenDirForSecondListWidgetClicked()</slot>
 </slots>
</ui>
